pages/docs/reference/javascript-dce.md

```diff
@@ -19,7 +19,8 @@ Unused declarations can appear in cases like:
   adapters for DOM, and so on. All of this functionality would require about 1.3 MB as a JavaScript file. A simple "Hello, world" application only requires
   console routines, which is only few kilobytes for the entire file.
 
-The Kotlin/JS Gradle plugin handles DCE automatically when you build a production bundle, for example by using the `browserProductionWebpack` task. Development bundling tasks don't include DCE.
+> The Kotlin/JS Gradle plugin handles DCE automatically when you build a production bundle, for example by using the `browserProductionWebpack` task. Development bundling tasks (like `browserDevelopmentWebpack`) don't include DCE.
```

Comment:
I don't think this should be a note. It's an important part of the general explanation. Notes are usually for some side additions that don't fit into the common flow.

Reply:
Thanks, I have removed the note tag from this, and added some bold emphasis to show the two differing cases instead.